.github/workflows/deep-diff.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
```

Comment:
Isn't this GH Action still required to flag edits to EN files that have existing translations?

Reply:
you're right. I brought back the gh action and changed the wording